src/sage/schemes/elliptic_curves/hom_velusqrt.py

```diff
@@ -1,7 +1,7 @@
 r"""
-√élu algorithm for elliptic-curve isogenies
```

Comment:
I think that the funny square-root character in place of V is intentional, as that is what @yyyyx4 and friends call the improved Velu algorithm 'velusqrt'.  So perhaps revert all these?

Reply:
Oh, I didn't know. I saw all of these (find them ugly and possibly insulting to Monsieur Vélu qui n'est pas un élu). I am personally against it in headers, as it just looks like a bug. If they want to use it as a curiosity in some of the documentation I am not against, it, but I would still want most of them to be reverted. Sorry to be spoil fun.

Reply:
@yyyyx4 How strongly do you feel about this?  Every time I see it I think there is something wrong with my computer screen!

Reply:
Well, √élu is the name given to the algorithm by its authors (not me). Other names that have appeared in the literature are all variants of "square‑root Vélu", with varying choices of punctuation, capitalization, accentuation, abbreviation, and word order. I obviously prefer √élu, but most of the other variants are acceptable to me. Identifying this algorithm with Vélu's seems wrong, though.

Reply:
Thanks for the comments. (Due to my bad choice of basing this issue on top of the other, the change of Vélu actually belongs to the dependency and not to this pull request. I will correct it there.)

I will of course correct this and I am sorry that I did this change without looking closer.
I will use "square-root Vélu". If there are other who prefer keeping it in the documentation and in the ``_repr_`` then they should comment here (or better on the other pull request). I don't have a strong feeling and I am ok to revert to √élu.